api/routes/project_insights.py

```diff
@@ -0,0 +1,52 @@
+from fastapi import APIRouter
+
+from api.utils.db import Connection, get_projectless_db_connection
+from db.python.layers.project_insights import ProjectInsightsLayer
+from models.models.project_insights import (
+    ProjectInsightsDetails,
+    ProjectInsightsSummary,
+)
+
+router = APIRouter(prefix='/project-insights', tags=['project-insights', 'summary', 'details', 'seqr', 'dashboard'])
+
+
+@router.post(
+    '/project-insights-summary',
+    operation_id='getProjectInsightsSummary',
+    response_model=list[ProjectInsightsSummary],
+)
+async def get_project_insights_summary(
+    project_ids: list[int] = None,
```

Comment:
Return `project_names: list[str]`, and then do map to IDs (which checks project access)